sdk/src/types/block/unlock/mod.rs

```
@@ -133,6 +134,14 @@ impl Unlocks {
     }
 }
 
+impl WorkScore for Unlocks {
+    fn work_score(&self, params: WorkScoreParameters) -> u32 {
+        self.iter()
+            .filter_map(|u| u.is_signature().then_some(params.signature_ed25519()))
```

Impl it for all and sum. This would silently fail when we add relevant unlock, it's already failing for Multi.

---

Done